Jenkinsfile

```diff
@@ -426,34 +435,28 @@ pipeline {
                             filename 'Dockerfile.mockbuild'
                             dir 'utils/rpms/packaging'
                             label 'docker_runner'
-                            additionalBuildArgs '--build-arg UID=$(id -u) --build-arg JENKINS_URL=' +
-                                                env.JENKINS_URL
+                            additionalBuildArgs '$BUILDARGS'
                             args  '--group-add mock --cap-add=SYS_ADMIN --privileged=true'
                         }
                     }
                     steps {
-                         githubNotify credentialsId: 'daos-jenkins-commit-status',
+                        githubNotify credentialsId: 'daos-jenkins-commit-status',
                                       description: env.STAGE_NAME,
                                       context: "build" + "/" + env.STAGE_NAME,
                                       status: "PENDING"
                         checkoutScm withSubmodules: true
-                        catchError(stageResult: 'UNSTABLE', buildResult: 'SUCCESS') {
+                        catchError(stageResult: 'UNSTABLE',
+                                   buildResult: 'SUCCESS') {
                             sh label: env.STAGE_NAME,
-                               script: '''rm -rf artifacts/centos7/
-                                          mkdir -p artifacts/centos7/
-                                          make CHROOT_NAME="epel-7-x86_64" -C utils/rpms chrootbuild'''
+                               script: 'ci/rpm_build.sh'
                         }
                     }
                     post {
                         success {
                             sh label: "Build Log",
```

Comment:
I wonder if we can put this whole block of steps (from 449-463) into a groovy step (is that what they are called?) in `pipeline-lib/vars`.

Reply:
We should be able to put all steps in a stage into groovy step.  I was planning on doing that in a separate PR.